src/component/organisms/g-menu.tsx

```diff
@@ -22,7 +22,7 @@ const GUIDE_HREF =
 
 type ChildMenuData = {
   label: string;
-  href: string;
+  href: string | null;
```

Comment:
```suggestion
  href?: string;
```

optional property는 물음표로 정의해주세요~